modules/virtual_network_gateway/.header.md

```diff
@@ -15,14 +15,13 @@ module "vng" {
   for_each = var.virtual_network_gateways
 
   name                = "${var.name_prefix}${each.value.name}"
-  location            = var.region
+  region              = var.region
   resource_group_name = local.resource_group.name
 
-  network   = each.value.network
-  subnet_id = module.vnet[each.value.vnet_key].subnet_ids[each.value.subnet_key]
+  ip_configurations = each.value.ip_configurations
+  instance_settings = each.value.instance_settings
+  subnet_id         = module.vnet[each.value.vnet_key].subnet_ids[each.value.subnet_key]
```

Comment:
I know it was like that before but after giving this a thought, I don't think we should reference modules or resources that are not a part of the snippet. Placeholder here too.